Week3/Assignment3/app/src/main/res/layout-land/activity_main.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <View
```

Comment:
This seems to work fine for the background color, there may be a background color property on the constraint layout itself so you wouldn't have to add an extra view if I'm not mistaken.